README.md

```diff
@@ -124,8 +124,8 @@ to `true`. Find it under `File > Preferences > Settings`.
 
 This extension almost exclusively uses the RLS for its feature support (syntax
 highlighting, snippets, and build tasks are provided client-side). The RLS uses
-the Rust compiler (rustc) to get data about Rust programs. It uses Cargo to
-manage building. Both Cargo and rustc are run in-process by the RLS. Formatting
-and code completion are provided by rustfmt and Racer, again both of these are
+the Rust compiler (`rustc`) to get data about Rust programs. It uses Cargo to
+manage building. Both Cargo and `rustc` are run in-process by the RLS. Formatting
+and code completion are provided by `rustfmt` and Racer, again both of these are
```

Comment:
I didn't put those in backticks because you can't type "Racer" at a command line and have it work, as opposed to rustc or rustfmt, which you can.

Elsewhere in the README backticks are used to denote something you can do (as in "`File > Preferences`") or something you can type (as in `"cd rust"`); this seemed to follow the same pattern.

Happy to amend if you'd rather run it differently, just lmk.

Reply:
@nrc Checking to see which you'd prefer:

* remove backtick modifications
* leave as-is

Reply:
With the exception of RLS and Cargo in "It uses Cargo to (...)", personally I think it's fine to use `lowercase-tool-name` everywhere else in this para, since every of these tools can be ran in the command line when talked about in this context (`tools` are run/provided by `tools`)